datasets/sondehub-telemetry.yaml

```diff
@@ -0,0 +1,55 @@
+Name: SondeHub Radiosonde Telemetry
+Description: >
+  SondeHub Radiosonde telemetry contains global radiosonde (weather balloon) data captured by SondeHub from our
+  participating radiosonde_auto_rx receiving stations. Currently 313 receiver stations are providing
+  data for an average of 384 radiosondes a day.
+
+  Data contains telemetry frames in Universal Sonde Telemetry format, including gps position, type,
+  altitude, pressure, temperature and humidity.
+Documentation: https://github.com/projecthorus/sondehub-infra/wiki/Amazon-Open-Data
+Contact: Michaela Wheeler <vk3fur@sondehub.org>
+ManagedBy: SondeHub
```

Comment:
Do you have a logo you would want to appear here too? You can email me it if so

Reply:
That's a good question! We don't have a logo specifically for sondehub yet but it's probably something we should work on for our new website. We usually use the radiosonde_auto_rx logo but I don't think it makes sense here so it's probably easier if we leave it blank for the moment.